Readme.md

```diff
@@ -133,6 +133,20 @@ Note: see https://github.com/thomseddon/node-oauth2-server/tree/master/examples/
  - *mixed* **error**
      - Truthy to indicate an error
 
+#### checkScope (accessToken, requiredScope, callback)
```

Comment:
What do you think about renaming the `requiredScope` variable to `scope`, so that it matches the `saveScope` function?